### ISSUE TITLE
Support configuration in bsconfig.json

### ISSUE THREAD

**Issue by nirvdrum** [baransu/graphql_ppx_re#40]
I've tried to change from graphql_ppx to graphql_ppx_re and ran into an issue where none of my configuration values provided in `bsconfig.json` are supported. I previously had used the following:

```
"ppx-flags": [
   ["graphql_ppx/ppx", "-verbose", "-apollo-mode", "-schema=src/graphql/schema.json"]
]
```

I see in the README that there are `GRAPHQL_PPX_APOLLO_MODE` and `GRAPHQL_PPX_SCHEMA` environment variables to support some of this functionality. However, it's considerably more convenient to have this as part of the checked-in source code where it can be managed per-project. Additionally, the BuckleScript build configuration docs [recommend not using environment variables for build configuration](https://bucklescript.github.io/docs/en/build-configuration.html#environment-variables).

If the ability to pass command-line arguments won't be reinstated, I think it'd be helpful to document how to migrate from `graphql_ppx`.

**Comment by baransu** [comment]
There is #16 which is about configuration file as in future, which many more options, ppx args will become hard to manage.

I'll leave this issue and try to fix current env variables solution as it's not the best as you pointed out.

**Comment by baransu** [comment]
https://github.com/ocaml-ppx/ocaml-migrate-parsetree/blob/master/MANUAL.md

**Comment by baransu** [comment]
Released as 0.5.0.
